js/os.ts

```diff
@@ -5,6 +5,14 @@ import * as util from "./util";
 import * as flatbuffers from "./flatbuffers";
 import { sendSync } from "./dispatch";
 
+/** process id */
+export let pid: number;
+
+export function setPid(pid_: number): void {
```

Comment:
This exposes a method that can be modified the PID

It should be unsafe.

Should we expose a method like ``function getPid(): number``?

Reply:
This isn't exposed to the deno module.